### ISSUE TITLE
Performance of FCN

### ISSUE THREAD

**Issue by Jongchan** [ZijunDeng/pytorch-semantic-segmentation#7]
Can the FCN code reproduce the numbers reported in the paper?
How much does it achieve in VOC and Cityscape?

**Comment by ZijunDeng** [comment]
Yes, the VOC performance of FCN has been reproduced, and you can see the result in *voc-fcn (caffe vgg). The [official FCN code](https://github.com/shelhamer/fcn.berkeleyvision.org) do not provide training setting of cityscapes and hence I spent little effort on it.

**Comment by Tgaaly** [comment]
what about cityscapes? I ran FCN8s for 150 epochs and got an accuracy of 0.80 but seemingly really bad other metrics. Is this expected? The only difference I made is reduce the batch size to 4 so it may take much longer than normal. It would be nice to know what the reproduced performances of all the models in this repo are!
```
[epoch 149], [val loss 80565.80459], [acc 0.79926], [acc_cls 0.29658], [mean_iu 0.22416], [fwavacc 0.68140]
best record: [val loss 79914.79803], [acc 0.80127], [acc_cls 0.29951], [mean_iu 0.22761], [fwavacc 0.68499]
```
